base/sparse/linalg.jl

```diff
@@ -946,17 +946,17 @@ function Base.cov(X::SparseMatrixCSC, vardim::Int=1; corrected::Bool=true)
     a, b = size(X)
     n, p = vardim == 1 ? (a, b) : (b, a)
 
-    # Cov(X) = E[(X-μ)'(X-μ)]
-    # = X'X - X'μ
+    # Cov(X) = E[(X-μ)(X-μ)'] = E(XX') - μμ'
 
     # Compute X'X using sparse matrix operations
     out = Matrix(Base.unscaled_covzm(X, vardim))
 
-    # Compute X'μ
+    # Compute (estimate of) nμ
     sums = sum(X, vardim)
+
+    # Subtract nμμ' from X'X
```

Comment:
It computes `X'X` or an estimate of `nE(XX')`. The notation is a little confusing since `X` is a vector in the population expression above: `E(XX')` whereas it here refers to the data matrix where each row is a vector similar to the original `X` vector. Hence `X'X` (where `X` is matrix) is an estimate of `nE(XX')` (where `X` is vector). The quantity `μ=EX` is the actual population mean (which is unknown) so we subtract an *estimate* of `nμμ'`. I included the *estimate* part in the comment above but decided it wasn't necessary to repeat here. I think people with a little stats background will understand the comment in the current form. Making it precise will either require explicit sum notation with subscripts which is verbose or require that we stop using the familiar population notation `Cov(X)=E(X-μ)(X - μ)'` and instead write the sample version in matrix form `1/(n-1)(X .- x̄)'(X .- x̄)` with `x̄ = mean(X, 1)` which is not how people typically express the covariance.